src/sentry/api/serializers/snuba.py

```diff
@@ -126,7 +126,7 @@ def value_from_row(row, tagkey):
 
 def zerofill(data, start, end, rollup):
     rv = []
-    start = ((int(to_timestamp(start)) // rollup) * rollup) - rollup
+    start = (int(to_timestamp(start)) // rollup) * rollup
```

Comment:
This will also change the bar graphs shown on issue list and issue details.

Reply:
@markstory I think this change is correct. 

I didn't see any Percy regression for the issue list and issue details page. But there were some snapshot changes for the Incidents pages.

I'm assuming any frontend components will need to be updated to compensate this change.